docs/source/api/deliveryservices_sslkeys_generate_letsencrypt.rst

```diff
@@ -0,0 +1,64 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-deliveryservices-sslkeys-generate-letsencrypt:
+
+*************************************************
+``deliveryservices/sslkeys/generate/letsencrypt``
+*************************************************
+
+``POST``
+========
+Generates an SSL certificate and private key using Let's Encrypt for a :term:`Delivery Service`
+
+:Auth. Required: Yes
+:Roles Required: "admin" or "operations"
+:Response Type:  Object (string)
+
+Request Structure
+-----------------
+:key:             The 'xml_id' of the :term:`Delivery Service` for which keys will be generated
+:version:         The version of the keys being generated
```

Comment:
this is the version of the certificate. every time a new cert is generated or added we increment the version to keep track of all of the certificates.  same as in deliveryservices_sslkeys_generate.rst

Reply:
Can you put something like that into the description? The only reason `deliveryservices_sslkeys_generate.rst` doesn't have a better description is because I didn't know what that field was when I was writing it.

Reply:
done. and i updated deliveryservices_sslkeys_generate.rst too